server/server.js

```diff
@@ -25,20 +23,67 @@ const getAvailableDates = function(propertyId, fromDate, toDate) {
       date: {
         [Op.between]: [fromDate, toDate],
       }
-      
+    }
+  });
+};
+
+const getRoomPrice = function(propertyId) {
+  return db.Property.findOne({
+    where: {
+      id: propertyId
     }
   });
 };
 
 app.get('/price/:propertyId', function(req, res) {
```

Comment:
I think you need to pass "next" as the third/last argument of the second arg here